samples/samples/autocommit.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import argparse
+
+from google.cloud.spanner_dbapi import connect
+
+
+def enable_autocommit_mode(instance_id, database_id):
+    """Enables autocommit mode."""
+    # [START spanner_enable_autocommit_mode]
+    from google.cloud.spanner_dbapi.checksum import ResultsChecksum
+
+    connection = connect(instance_id, database_id)
+    connection.autocommit = True
+    print("Autocommit mode is enabled.")
+
+    cursor = connection.cursor()
+    cursor._checksum = ResultsChecksum()
```

Comment:
Why does the user need to initialise a private variable? Can this not be initialised as part of the cursor factory?

Reply:
Thank you for the review! And sorry for the delayed answer.
I've made an update and placed this part in the test.

Reply:
This is not quite what I meant. The sample now looks as I would expect but I'm unclear as to why you still need to set `cursor._checksum` in the test. Is there a reason that [`_checksum` defaults to `None`](https://github.com/googleapis/python-spanner/blob/master/google/cloud/spanner_dbapi/cursor.py#L61)? Could it be updated to `ResultsChecksum()` instead?